src/features/tracks/lib/type-guards.ts

```diff
@@ -0,0 +1,9 @@
+import { SortOrder, SortValue } from './types';
+
+export const isValidSortValue = (value: string): value is SortValue => {
+  return ['title', 'artist', 'album'].includes(value);
+};
+
+export const isValidSortOrder = (value: string): value is SortOrder => {
+  return ['asc', 'desc'].includes(value);
+};
```

Comment:
Great that you added type guards, but values in an array are hardcoded. 
It's better to move these arrays into constants and depending on them create types `SortValue` and `SortOrder`